app/Http/Controllers/Api/V1/GroupController.php

```diff
@@ -80,6 +80,33 @@ public function index(Organization $suborganization)
         ], 200);
     }
 
+    /**
+     * Get All Organization Groups
+     *
+     * @urlParam suborganization required The Id of a suborganization Example: 1
+     * Get a list of the groups of an organization.
```

Comment:
Description comments come under the title comments like shown in docs
https://beyondco.de/docs/laravel-apidoc-generator/getting-started/documenting-your-api#grouping-endpoints